### DISCUSSION TITLE
Clickable/Linkable Metrics

### DISCUSSION THREAD

**Discussion by piljac1** [laravel/nova-issues#4287] [Ideas & Feature Requests]
It would be great if you could add a link to the metrics displayed count and/or a button to execute a specific action (either a litteral Nova action, navigate to a route/lens, apply specific filters, open an external link, etc.).

Metrics are a great feature and are well loved by our clients, but all of them find it lackluster that you cannot click on the metrics to see more information.

For example, we implemented a metric indicating the number of unsubscribed users. However, our clients would like if you could click on the metric and get redirected to the users index filtered on the unsubscribed ones.

That's one of many metrics that would benefit from that feature. 

Thanks in advance for your consideration!

**Comment by ianrobertsFF** [comment]
It would be handy. but in the mean time `Metric` just extends `Card` on the php side of things, so you could create an asset, extend the metric vue files and set `public $component` in your Metric to match the new component name.

**Reply by piljac1** [reply]
That's an option. Just figured I would post the idea publicly because we have to implement it in one of our next sprints. So if the idea gathers a lot of positive attention and the Nova team decides the implement a solution before we have to implement it on our end, it's a win for us and the rest of Nova users! :)

**Comment by sutcliffe121** [comment]
Agreed. Would love to be able to add a link/url to a metric. It can be achieved by creating a custom card, but more work. Seems so logical that users would want to click a metric to look at detail of the resource.

**Comment by cweiske** [comment]
There is the external [nova-linkable-metrics package](https://github.com/saintsystems/nova-linkable-metrics) that can be used, but I would also prefer a native solution.
